src/lib/const.ts

```diff
@@ -434,7 +419,10 @@ data WorkerServices:
 func showSubnet() -> []WorkerServices:
```

Comment:
this function shouldn't filter out empty workers, I think.

Reply:
imporoved the function so at least host_id is returned